src/components/pages/Home/Onboarding/Header.module.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  height: var(--wrapper-height);
+  padding-left: 0;
+}
+
+.content {
+  padding: var(--spacer);
+}
+
+.subtitle {
+  display: inline-block;
+  padding: calc(var(--spacer) / 8) calc(var(--spacer) / 4) 0;
```

Comment:
nitpick(blocking): this looks like I need to think a lot about padding and why these numbers were chosen. Add a comment explaining why exactly these values are needed or use variable names.

Reply:
We took the style from the component inside the hero section to keep it consistent, so I'm not sure about the reason of these specific padding combination. We could probably extract them into a separate component to keep the style consistent over the page